src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/MigrationJobScaffoldingTest.kt

```diff
@@ -13,7 +13,7 @@ class MigrationJobScaffoldingTest : SeedTestBase() {
       .uri("/migration-job")
       .bodyValue(
         MigrationJobRequest(
-          jobType = MigrationJobType.updateAllUsersFromCommunityApi,
+          jobType = MigrationJobType.allUsersFromCommunityApi,
```

Comment:
I don't know why every time I make changes to the migration jobs, the generator changes the name